apps/sensenet/src/components/localization/localization.tsx

```diff
@@ -1,16 +1,17 @@
 import React from 'react'
-import { Typography } from '@material-ui/core'
 import clsx from 'clsx'
 import { SimpleList } from '../content/Simple'
 import { useGlobalStyles } from '../../globalStyles'
+import { useLocalization } from '../../hooks'
 
 export default function Localization() {
   const globalClasses = useGlobalStyles()
+  const localizationDrawerTitles = useLocalization().drawer.titles
 
   return (
     <div className={globalClasses.contentWrapper}>
       <div className={clsx(globalClasses.contentTitle, globalClasses.centeredVertical)}>
-        <Typography variant="h6">Localization</Typography>
+        <span style={{ fontSize: '20px' }}>{localizationDrawerTitles.Trash}</span>
```

Comment:
I think h6 is fine here.
```suggestion
        <span style={{ fontSize: '20px' }}>{localizationDrawerTitles.Localization}</span>
```